src/constants/pets.js

```diff
@@ -1431,6 +1431,48 @@ module.exports = {
         }],
       },
     },
+    GRANDMA_WOLF: {
+      name: 'Grandma Wolf',
+      statModifiers: {
+        health: 1,
+        strength: 0.25,
+      },
+      abilityModifiers: {
+        0: {
+          mf1: [0.02, 0.02, 0.02, 0.02, 0.02],    // Math.floor((8 + this.level * 0.02) * 10) / 10
```

Comment:
You can actually use a single number for these since they remain constant at all levels